### ISSUE TITLE
Licencing?

### ISSUE THREAD

**Issue by broomfn** [riderx/capacitor-updater#7]
Hi,

This looks like a fantastic library, but the AGPL licence seems a little prescriptive, correct me if I'm wrong, but If I include your npm package I would need to open source the rest of my Ionic mobile application?

Thanks

Neil

**Comment by riderx** [comment]
Hey @broomfn thanks for the feedback, i understand your issue.
I use AGPL as explained here : https://opensource.com/article/17/1/providing-corresponding-source-agplv3-license
```
What is often misunderstood is that the source code requirement in AGPLv3 Section 13 is triggered only where the AGPLv3 software has been modified by "you" (for example, the entity providing the network service). My interpretation is that, so long as "you" do not modify the AGPLv3 code, the license should not be read as requiring access to the Corresponding Source in the manner prescribed by Section 13. As I see it, many unmodified and standard deployments of software modules under AGPL simply do not trigger Section 13, although making the source code available even if not required under the license is a good idea.
```
Include it in your project don't force you to opne source it.
this is only needed if you modify `capacitor-updater`

**Comment by broomfn** [comment]
@riderx Many thanks for the clarification.

I know from the past there have been many arguments as to what defines a derived work, so just wanted to be clear before we use it, if you're happy we don't need to open source it that's good enough for me.

**Comment by riderx** [comment]
Yes of course, it's a long debate, i should have writed it, on the repo, i know that unclear for some people

**Comment by riderx** [comment]
After some conversation in the discord and the Discover of this:
https://opensource.stackexchange.com/questions/6879/may-i-use-agpl-license-in-a-desktop-application-without-providing-the-source-cod
It seems pretty obvious now AGPL3 is not what i think, so i migrated to LGPL3

**Comment by riderx** [comment]
After other feedback from users MPL seems better designed for plugin code
